### PR TITLE
upgrade source code compatibility JDK11

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

**Comment by Xiaochao8** [comment]
Which way are you applying?
image

**Comment by tigershi** [comment]
截屏2023-01-17 11 31 44

**Review by linr211 — APPROVED** [review]

**Review by Xiaochao8 — APPROVED** [review]

File: g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/LocaleUtils.java [hunk 1]
```diff
@@ -27,12 +25,12 @@ public class LocaleUtils {
 	 * @return true if the format is fine
 	 */
 	public static boolean isLanguageTag(String languageTag) {
-		ParseStatus sts = new ParseStatus();
-		LanguageTag.parse(languageTag, sts);
-		if (sts.isError()) {
+		try {
```
Comment (Xiaochao8) [inline g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/LocaleUtils.java:28]: Why is this changed needed?

Comment (tigershi) [reply g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/LocaleUtils.java:28]: it use the other module of jdk11, when start  JAVA VM
